docs/reference/advanced_configuration.rst

```diff
@@ -9,7 +9,7 @@ Full configuration options:
 
     .. code-block:: yaml
 
-        # app/config/config.yml
+        # config/packages/symfony_cmf_routing_extra.yaml
```

Comment:
this should be cmf_routing.yaml.

i notice that the part below is really outdated. the very first experiments had this config key, but since then we changed to `cmf_routing`. routers_by_id is then correct, however.

Reply:
the service id of the dynamic router in the comment below is cmf_routing.dynamic_router

Reply:
thank you, I updated it, can you please verify again?